docker-hub/builds/advanced.md

```diff
@@ -74,6 +73,8 @@ The following hooks are available:
 * `hooks/pre_push` (only used when executing a build rule or [automated build](index.md) )
 * `hooks/post_push` (only used when executing a build rule or [automated build](index.md) )
 
+A hook will need to return a 0 exit code to let the next ones be executed; a different exit code will stop the build and will be marked as failed by the system.
```

Comment:
```suggestion
A hook will need to return a 0 exit code to let the next ones be executed. A different exit code will stop the build and it will be marked as failed by the system.
```